yoti-sdk-api/src/main/java/com/yoti/api/client/Anchor.java

```diff
@@ -3,6 +3,10 @@
 import java.security.cert.X509Certificate;
 import java.util.List;
 
+/**
+ * Anchors represent where, when and how a profile {@link Attribute} value was acquired or verified.  For example, Attributes sourced from a backing
```

Comment:
do you mind to add a `,` here `Anchors represent where, when, and how....`? Also, there is one more space after the first sentence.